src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -512,6 +529,15 @@ public void ProcessCommand(IExecutionContext context, string inputLine, ActionCo
             command.Properties.TryGetValue(IssueCommandProperties.Line, out string line);
             command.Properties.TryGetValue(IssueCommandProperties.Column, out string column);
 
+            if (ActionCommandManager.EnhancedAnnotationsEnabled(context)) 
```

Comment:
do we need to check this again? we already skip this in `TryProcessCommand()`

Reply:
So `WarningCommandExtension` and `ErrorCommandExtension` both inherit from this class and are processed. There are 2 pieces that are under the same feature flag: The `notice` command and the additional field support.